bundles/action/org.openhab.action.slack/src/main/java/org/openhab/action/slack/internal/Slack.java

```diff
@@ -0,0 +1,106 @@
+/**
+ * Copyright (c) 2010-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.action.slack.internal;
+
+import java.io.IOException;
+
+import org.openhab.core.scriptengine.action.ActionDoc;
+import org.openhab.core.scriptengine.action.ParamDoc;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.ullink.slack.simpleslackapi.SlackChannel;
+import com.ullink.slack.simpleslackapi.SlackSession;
+import com.ullink.slack.simpleslackapi.SlackUser;
+import com.ullink.slack.simpleslackapi.impl.SlackSessionFactory;
+
+/**
+ * This class provides static methods that can be used in automation rules
+ * for sending messages to Slack.
+ *
+ * It uses the Simple Slack Api from https://github.com/Ullink/simple-slack-api
+ *
+ * @author Rino van Wijngaarden (rino.van.wijngaarden@gmail.com)
+ *
+ */
+public class Slack {
+
+    private static final Logger logger = LoggerFactory.getLogger(Slack.class);
```

Comment:
The new convention in ESH/OS2 is to not have static loggers, but all the other 1.x add-ons have static loggers, so I would say this is OK.